### PR TITLE
[bugfix]fix calculation of weighted loss

### PR DESCRIPTION
**Pull request by eric-gecheng**

### CODE REVIEW

File: tzrec/models/match_model.py [hunk 1]
```diff
@@ -233,7 +233,9 @@ def _loss_impl(
             label = _zero_int_label(pred)
         losses[loss_name] = self._loss_modules[loss_name](pred, label)
         if self._sample_weight:
-            losses[loss_name] = torch.mean(losses[loss_name] * sample_weight)
+            losses[loss_name] = torch.sum(
```
Comment (tiankongdeguiji) [inline tzrec/models/match_model.py]: use torch.mean(xxx)/torch.mean(xxx) is better, loss and sample weight could be too large

**Review by tiankongdeguiji — CHANGES_REQUESTED** [review]

**Review by tiankongdeguiji — APPROVED** [review]
LGTM
